File: exercism/python/archive/leap/leap.py
""" module to calculate whether this is a leap year """
# -*- coding: utf-8 -*-

from __future__ import unicode_literals

def is_leap_year(year=0):
    """ method to check for the leap year """
    if year == 0:
        return False

    if year % 4 == 0 and (year % 100 != 0 or year % 400 == 0):
        return True

    return False

File: exercism/go/archive/leap/leap.go
// Package leap helps us calculate whether the input is a leap year
package leap

// testVersion should match the targetTestVersion in the test file.
const testVersion = 2

// IsLeapYear is great for telling us whether the int year is a leap (return bool)
func IsLeapYear(year int) bool {
	if year == 0 {
		return false
	}

	if ((year % 4) == 0) && (((year % 100) != 0) || ((year % 400) == 0)) {
		return true
	}

	return false
}
